tests/run.go

```diff
@@ -127,6 +125,20 @@ var knownFails = map[string]failReason{
 	"fixedbugs/issue30977.go": {category: neverTerminates, desc: "does for { runtime.GC() }"},
 	"fixedbugs/issue32477.go": {category: notApplicable, desc: "uses runtime.SetFinalizer and runtime.GC"},
 	"fixedbugs/issue32680.go": {category: notApplicable, desc: "uses -gcflags=-d=ssa/check/on flag"},
+
+	// These are new tests in Go 1.13-1.16.
+	"fixedbugs/issue19113.go":  {category: lowLevelRuntimeDifference, desc: "JavaScript bit shifts by negative amount don't cause an exception"},
```

Comment:
Is this sort of thing worth mentioning in the compat table?

Reply:
Yeah, this is a good idea, although I am again torn about where to put that note. docs/packages.md is supposed to be about packages and as a user I wouldn't expect to find runtime compatibility information there. On the other hand, creating a third document just for a few paragraphs' sake… Any thoughts?

Reply:
Maybe renaming packages.md into compatibility.md and putting all the bits together is a reasonable way to go, although I expect this will break a bunch of links, some of which are external, which is not great.

Reply:
Actually, I think I'll do this: keep packages.md and compiler.md with their narrow-scope purpose and create a more general compatibility.md as a central doc for this kind of thing, linking out to other docs when appropriate. After a bit of thought there's merit in describing the high-level state of things, as well as low-level details like that. Since it'll deserve its own review, I'll do that in a separate PR.